.github/workflows/deploy-production.yml

```
@@ -3,31 +3,238 @@ name: Deploy Production
+      - name: Install Vercel CLI
+        run: npm install --global vercel@latest
+
+      - name: Build and Deploy API
+        env:
+          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+          VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+          VERCEL_PROJECT_ID: ${{ secrets.VERCEL_API_PROJECT_ID }}
+          NEXT_PUBLIC_WEB_URL: ${{ secrets.NEXT_PUBLIC_WEB_URL }}
+          NEXT_PUBLIC_ADMIN_URL: ${{ secrets.NEXT_PUBLIC_ADMIN_URL }}
+          MOCK_USER_ID: ${{ secrets.MOCK_USER_ID }}
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:1:52: Double quote to prevent globbing and word splitting

(shellcheck)